{"status":"FAIL","msg":"FAIL:\n TypeError: Expected an object, found Number(42.0)\n    at ../../test262/test/built-ins/TypedArray/prototype/map/BigInt/arraylength-internal.js:18\n    at anonymous (../../test262/harness/testBigIntTypedArray.js:34)\n    at testWithBigIntTypedArrayConstructors (../../test262/test/built-ins/TypedArray/prototype/map/BigInt/arraylength-internal.js:17)\n\n","path":"../../test262/test/built-ins/TypedArray/prototype/map/BigInt/arraylength-internal.js"}